index.d.ts

```diff
@@ -133,6 +136,42 @@ declare namespace camelcaseKeys {
 		*/
 		readonly stopPaths?: readonly string[];
 
+		/**
+		Exclude children at the given object key from being camel-cased.
```

Comment:
I would make it clear that this matches any object key with the given names at any nesting level.

Reply:
done